### PR TITLE
Add sentry

### PR DESCRIPTION
**Pull request by anxolin**
Similarly to https://github.com/gnosis/cowswap/pull/1256 and https://github.com/gnosis/cowswap/pull/1258, adds sentry to the explorer too

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ env:
   APP_ID: 1
   REPO_NAME_SLUG: gpui
   PR_NUMBER: ${{ github.event.number }}
+  REACT_APP_SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
```
Comment (alfetopito) [inline .github/workflows/ci.yml:16]: Is it on repo settings yet?

**Review by alfetopito — APPROVED** [review]
Same questions as Cowswap's PR

File: .github/workflows/ci.yml (hunk 1 above)
Comment (anxolin) [reply .github/workflows/ci.yml:16]: Yes! it is

**Comment by anxolin** [comment]
> Same questions as Cowswap's PR

Same answers :)

**Pull request merged**